controllers/drplacementcontrol_controller_test.go

```diff
@@ -735,10 +735,23 @@ func getLatestDRPC() *rmn.DRPlacementControl {
 }
 
 func clearDRPCStatus() {
-	latestDRPC := getLatestDRPC()
-	latestDRPC.Status = rmn.DRPlacementControlStatus{}
-	err := k8sClient.Status().Update(context.TODO(), latestDRPC)
-	Expect(err).NotTo(HaveOccurred())
+	localRetries := 0
+
+	for localRetries < updateRetries {
+		latestDRPC := getLatestDRPC()
+		latestDRPC.Status = rmn.DRPlacementControlStatus{}
+
+		err := k8sClient.Status().Update(context.TODO(), latestDRPC)
+		if errors.IsConflict(err) {
+			localRetries++
```

Comment:
This is a good place to log that about the retry, something like:

    Conflict updating drpc, retrying...

Reply:
Lets do this later, more code need this logging.